app/assets/javascripts/components/Dashboard.jsx

```diff
@@ -0,0 +1,19 @@
+class Dashboard extends React.Component {
+  render() {
+  	const { Tabs, Tab } = ReactBootstrap;
+  	const { user } = this.props;
+    return (
+    	<div>
+	    	<DashboardHeader name={user.first_name}/>
+	    	<Tabs defaultActiveKey={1} id="user-dashboard">
+	    		<Tab eventKey={1} title="Tasks">
+	    			<TaskList userId={user.id} />
+	    		</Tab>
+	    		<Tab eventKey={2} title="Clients">
+	    			<ClientList />
+	    		</Tab>
+	    	</Tabs>
+	    </div>
+  	);
+  }
+}
```

Comment:
nit: newline

Reply:
changed in: https://github.com/calblueprint/sjc/pull/14/commits/185ffe235d12ea28170ab87c0c1430389da5af3e